api/src/main/kotlin/team/backend/domain/dto/QuizCommand.kt

```diff
@@ -0,0 +1,14 @@
+package team.backend.domain.dto
+
+class QuizCommand {
```

Comment:
How about separating the package into command and query?

Reply:
- command (package)
  - xxxCommand
- query (package)
  - xxxQuery

Reply:
I applied package separation.